From e54a9303bd863a5fb8728925eab5d524cc216429 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Thu, 2 Feb 2012 17:12:44 +0100
Subject: [PATCH] Return non-zero exit code on tests failure

---
 regression/TEST |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/regression/TEST b/regression/TEST
index bab2158..c13843e 100755
--- a/regression/TEST
+++ b/regression/TEST
@@ -415,6 +415,7 @@ then
   cat  $ERR   
   echo '****  Errors occured: ./output not removed'
   echo
+  exit 1
 else
   cd ..
   rm -rf output
-- 
1.7.7.6

